libraries/rating-bar/src/main/java/com/trendyol/uicomponents/ratingbar/Extensions.kt

```diff
@@ -0,0 +1,31 @@
+package com.trendyol.uicomponents.ratingbar
```

Comment:
this file can be moved to **uicomponents** package.

- [ ] move file

Reply:
If I move these extension functions into another package, they can't be internal. Will be accessible on project.

Reply:
You're right! When new component will be developed, we can create core-module to prevent same extensions.

Reply:
We will check that later as you said. Also we may use another extensions library that provides all our needs.